src/sentry/api/endpoints/organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,137 @@
+from collections import namedtuple
+from datetime import datetime, timedelta, timezone
+
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+from urllib3 import Retry, connection_from_url
+
+from sentry import features
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.serializers import serialize
+from sentry.api.utils import get_date_range_from_params
+from sentry.snuba.discover import timeseries_query
+from sentry.utils import json
+
+ads_connection_pool = connection_from_url(
```

Comment:
@nirajdewani We have a wrapper on `connection_from_url()` in https://github.com/getsentry/sentry/blob/c48d89f749902149cf95fc33e0cad70c0f60a084/src/sentry/net/http.py#L196-L199

Can you use that instead? 

It can be imported with `from sentry.net.http import connection_from_url`